config/sidekiq.yml

```diff
@@ -1,8 +1,6 @@
-:verbose: false
-#:logfile: ./log/sidekiq.log
-:pidfile: ./tmp/pids/sidekiq.pid
-:concurrency: <%= ENV["SIDEKIQ_CONCURRENCY"] || 10 %>
-:timeout: 30
+:verbose: <%= ENV.fetch('SIDEKIQ_VERBOSE', false) %>
```

Comment:
Does this need a `.to_bool` or anything?

Reply:
```suggestion
:verbose: <%= !!ENV.fetch('SIDEKIQ_VERBOSE', false) %>
```